### DISCUSSION TITLE
Qt Creator Setup

### DISCUSSION THREAD

**Discussion by erik-flick** [strawberrymusicplayer/strawberry#1616] [Q&A] [answered]
Heya.  I'm a newbie Qt developer and I'm wondering how I can get Strawberry up and running with Qt Creator on Windows 10.  I'm used to importing a project file, but there's not one included with the repo.  Is development not meant to be in Qt Creator?

Thanks!

**Comment by jonaski** [comment, marked as answer]
You need to add a kit in Qt Creator which points the Qt installation also containing all the other libraries, see https://github.com/strawberrymusicplayer/strawberry-msvc
Once that's setup you just open the `CMakeLists.txt` file in Qt creator
